zerver/tests/test_digest.py

```diff
@@ -144,7 +149,19 @@ def test_enough_traffic(self, mock_send_future_email: mock.MagicMock) -> None:
         ) as enough_traffic_mock:
             bulk_handle_digest_email([othello.id], in_the_future)
             mock_send_future_email.assert_called()
-            enough_traffic_mock.assert_called_once_with([], 0)
+            enough_traffic_mock.assert_called_once_with([], 0, 0, False)
```

Comment:
Yeah, see here even the test shows that we're not passing a string for hot conversations.